sdk/samples/iot/paho_iot_hub_telemetry_sample.c

```diff
@@ -168,12 +168,13 @@ static void send_telemetry_messages_to_iot_hub(void)
           rc = az_iot_hub_client_telemetry_get_publish_topic(
               &hub_client, NULL, telemetry_topic_buffer, sizeof(telemetry_topic_buffer), NULL)))
   {
-    LOG_ERROR("Failed to get the Telemetry publish-topic: az_result return code 0x%08x.", rc);
+    IOT_SAMPLE_LOG_ERROR("Failed to get the Telemetry topic: az_result return code 0x%08x.", rc);
     exit(rc);
   }
 
-  const char* telemetry_message_payloads[MAX_TELEMETRY_MESSAGE_COUNT] = {
-    "Message One", "Message Two", "Message Three", "Message Four", "Message Five",
+  char const* telemetry_message_payloads[MAX_TELEMETRY_MESSAGE_COUNT] = {
```

Comment:
curious why you changed these

Reply:
Now when I print out the payload for the message, it looks like json formatted telemetry.  I was trying to mimic the telemetry format found in twin sample.